client/src/components/UserCard/UserCard.tsx

```diff
@@ -0,0 +1,62 @@
+import {
+  Grid,
+  Paper,
+  Typography,
+  Box,
+  Card,
+  TextField,
+  Button,
+  CardActions,
+  CardContent,
+  Divider,
+  CardMedia,
+  CardActionArea,
+  Avatar,
+  Input,
+} from '@material-ui/core';
+import { Rating } from '@material-ui/lab';
+
+import { Link } from 'react-router-dom';
+
+import useStyles from './useStyles';
+const profileImg = '775db5e79c5294846949f1f55059b53317f51e30.png';
```

Comment:
I'd recommend renaming image files to something meaningful and descriptive of the image it contains 👍🏼

Reply:
> > I'm not certain what the best way to implement the date range search would be, I've attempted for a few hours to map over the dates. I have some of the code commented out that may be useful for further integrating the date search. For now, I figure it's best if we skip the date search and merge this in, fixing date search in a hotfix if needed as presentations are soon and availability isn't fully merged. Let me know if there are any implementation ideas.
> 
> One idea would be to count the overlapping days - for example, you can iterate over the requested date range, see if a sitter is available at that day and count it towards your total if they are. Then you can display the profile if it's an >80% match.

Would this filter function be more important than stripe API at the moment? Trying to decide where to dedicate my time. 

> > I'm not certain what the best way to implement the date range search would be, I've attempted for a few hours to map over the dates. I have some of the code commented out that may be useful for further integrating the date search. For now, I figure it's best if we skip the date search and merge this in, fixing date search in a hotfix if needed as presentations are soon and availability isn't fully merged. Let me know if there are any implementation ideas.
> 
> One idea would be to count the overlapping days - for example, you can iterate over the requested date range, see if a sitter is available at that day and count it towards your total if they are. Then you can display the profile if it's an >80% match.

so something like:
for day in request {
    weekday = moment(day).format('dddd');
    if day[weekday].available == false{
        counter ++ 
}}
numberOfDays = request.length
percent = counter * days/ 100  
if percent <= 80{
return profile 
}

The issue I came across with that method was using 'day' as a key, it wasn't working as it's a string. Is there a way for me to force the type?

Reply:
What type does it expect instead? I'm sure there is a suitable conversion somewhere

Reply:
I have it fully working now, no issue with the filter functionality!